harness/determined/cli/dev.py

```diff
@@ -348,6 +358,11 @@ def call_bindings(args: argparse.Namespace) -> None:
                         "call a function from bindings",
                         [
                             cli.Arg("name", help="name of the function to call"),
+                            cli.Arg(
+                                "--skip-auto-picking",
```

Comment:
The cli functionality here isn't to "skip" auto picking. It actually forces auto picking.
So I'd rename this arg to `auto-pick` or `skip-picking`

Reply:
Though those names would also imply that you'd be force picking out of multiple hits.
Since you want it to match some regex, I think the best flag would be `--use-match`. This seems like a clearer description of what you want it to do. And if there are multiple matches, an extra step for disambiguation still makes sense.